### ISSUE TITLE
Function names in stack traces off by 1

### ISSUE THREAD

**Issue by marvinhagemeister** [babel/babel#9883]
## Bug Report

**Current Behavior**

The function names inside a stacktrace are off by one. The first function name is repeated and thus printed tiwce:

```txt
Error:
    at foo (/my-project/src/foo.js:5:15)
    at foo (/my-project/src/foo.js:9:3)
    at bar (/my-project/src/foo.js:13:3)
    ...more
```

The second function name should be `bar` and the third should be `bob`. The individual line numbers and columns are correct though, it's just the function names that are shifted by 1.

**Input Code**

[Link to repo](https://github.com/marvinhagemeister/babel-sourcemap-bug)

```js
function foo() {
  console.log(new Error().stack);
}

function bar() {
  foo();
}

function bob() {
  bar();
}

bob();
```

**Expected behavior/code**

When the script is executed, the stack trace should be similar to this:

```txt
Error:
    at foo (/my-project/src/foo.js:5:15)
    at bar (/my-project/src/foo.js:9:3)
    at bob (/my-project/src/foo.js:13:3)
    ...more
```
A clear and concise description of what you expected to happen (or code).

**Babel Configuration (.babelrc, package.json, cli command)**

This issue happens even without any plugins. The config is therefore empty.

```js
{}
```

**Environment**
- Babel version(s): 7.4.3
- Node/npm version: 11.7.0
- OS: Debian Linux
- Monorepo: no
- How you are using Babel: `cli` or `loader`. Happens with both

**Possible Solution**

No clue

**Additional context/Screenshots**
Over at preactjs we're in the process of writing our adapter for the next generation of react-devtools. It allows us to display a tree of hooks (plain functions). Because there is no way to retrieve the parent function (for good reason) both the react team und we at preact rely on stack traces.

This could be related to #6659 .

**Comment by JLHwung** [comment]
This is a bug of `source-map-support`: https://github.com/evanw/node-source-map-support/pull/253

Close it as it is out of babel scope.

**Comment by LinusU** [comment]
This should be fixed in `source-map-support` `0.5.14` :shipit:
